gatsby/src/components/SiteDistinction/SiteDistinction.module.scss

```diff
@@ -0,0 +1,135 @@
+@use '@nice-digital/nds-core/scss/core';
+@import '../../styles/settings';
+
+.wrapper {
+  border-top: core.rem(core.$nds-spacing-x-small) solid;
+  position: sticky;
+  top: -0.1px; // Hack to avoid 1px gap due to some weird Chrome sub pixel rounding
+  z-index: 1;
+
+  &.bnf {
+    border-top-color: $bnf-brand-colour;
+  }
+
+  &.bnfc {
+    border-top-color: $bnfc-brand-colour;
+  }
+}
+
+.container {
+  position: relative;
+}
+
+.tabs {
+  margin: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.currentSiteTab,
+.otherSiteTab {
+  @include core.nds-font-size(-1);
+  align-items: center;
+  appearance: none;
+  background: none;
+  border: 0;
+  color: core.$nds-colour-true-black;
+  display: flex;
+  flex-direction: row;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  gap: core.rem(core.$nds-spacing-x-small);
+  justify-content: center;
+  letter-spacing: 1px;
+  line-height: 1.5rem;
+  padding: core.rem(core.$nds-spacing-small);
+  position: relative;
+  text-decoration: none;
+  text-shadow: 1px 0 0 currentColor;
+  top: -1px; // To avoid a 1px gap in Chrome
+
+  &:hover {
+    color: core.$nds-colour-true-black;
+  }
+}
+
+.currentSiteTab {
+  z-index: 1; // To site above the other site tab
```

Comment:
sit above or site above? Either works I guess...?

Reply:
Should be sit!